packages/opentelemetry-core/test/context/HttpTraceContext.test.ts

```diff
@@ -78,6 +78,23 @@ describe('HttpTraceContext', () => {
       );
       assert.deepStrictEqual(carrier[TRACE_STATE_HEADER], 'foo=bar,baz=qux');
     });
+
+    it('should ignore invalid span context', () => {
+      const spanContext: SpanContext = {
```

Comment:
you could use `api.INVALID_SPAN_CONTEXT` here - or at least `INVALID_SPANID` and `INVALID_TRACEID`